### DISCUSSION TITLE
about configs and textures

### DISCUSSION THREAD

**Discussion by nikodiseas** [jetelain/ArmaRealMap#87] [General]
hi. I recently asked how to adapt textures not from this mod so that they look correct, and not a huge black shadow. at the moment I have figured out how to adapt vanilla (Mediterranean and Tanoa), I have already created maps of the Sonoran desert, the Greek island of Rhodes, a bit of Papua, all as identical as possible... I just don't understand yet how to do this with textures from other mods, for example, I really like what is in Tobruk, and I would like to use it for Egypt. in this regard, the question arises how to adapt textures from other mods. to do this, where to look for the necessary configs, or how to create, in general, how to interact with them? thank you so much for this program! In the end, I finally realized a good half of my desires throughout the long-term Arma game...

**Comment by nikodiseas** [comment]
it's just that as far as I understand, the necessary tools will not appear in the coming days, and the brain wants to be implemented right now xD

**Comment by jetelain** [comment]
Hi,
Textures is a complex matter in Arma 3, because it has some configuration alongside the map to work properly. You should be able to get the texture by enabling the corresponding mods in the configuration, but you wont have any "Clutter" on it (small 3d objects like grass).
I plan to add tooling for that, but it will takes times.
